### DISCUSSION TITLE
Adding a downloadable to a tag

### DISCUSSION THREAD

**Discussion by guivnf** [squidfunk/mkdocs-material#6651] [Q&A] [answered]
so i wanted to know how i can add an example that can be downloaded through a tag simmilarly to the one in https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#tag-icons-and-identifiers

To be more exact, this:
![image](https://github.com/squidfunk/mkdocs-material/assets/77132244/18e2278f-069e-45a8-baef-a59c57d438d5)
If anyone can link me to where this is documented or if its not documented how can i do this. I'd be willing to sign up for insiders if needed too, just wanna know how i can do it...

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @MetalKnight56,
this functionality is custom to the Material documentation. It's currently not exposed in any straightforward way.
However, you can replicate it on your own https://github.com/squidfunk/mkdocs-material/discussions/6544#discussioncomment-7901513
The `<!-- md:example tags-with-icons -->` shortcode is processed via the `shortcodes.py` hook:
https://github.com/squidfunk/mkdocs-material/blob/ab5ccbe6b8c49dbf58a25eb066a98b243a382967/src/overrides/hooks/shortcodes.py#L57
and replaced with the appropriate content.

**Reply by guivnf** [reply]
Thank you, appreciatte the help :)
